low-level/framework/sspl_ll_d

```diff
@@ -810,6 +811,7 @@ if __name__ == "__main__":
         main(conf_reader, systemd_support)
 
     except (Exception) as err:
+        traceback.print_exc()
```

Comment:
not good to be seen in generic logs, add under debug log if you need

Reply:
Done. Now its only in logger.debug